client/packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -68,10 +68,12 @@ const getColumnLookup = <T extends RecordWithId>(): Record<
     key: 'expiryDate',
     label: 'label.expiry',
     width: 100,
-    formatter: dateString =>
-      dateString
+    formatter: dateString => {
```

Comment:
offroad - the date formatter won't show `[multiple]` since that isn't a valid date. Side note: have to fix the hard coded [multiple]